debezium-core/src/main/java/io/debezium/relational/RelationalSnapshotChangeEventSource.java

```diff
@@ -429,6 +429,14 @@ protected ChangeRecordEmitter getChangeRecordEmitter(SnapshotContext<O> snapshot
         return overriddenSelect != null ? Optional.of(enhanceOverriddenSelect(snapshotContext, overriddenSelect, tableId)) : getSnapshotSelect(snapshotContext, tableId);
     }
 
+    /**
+     * Returns a valid query string for the specified tableId
+     *
+     * @param tableId the table to generate a query for
+     * @return a valid query string with excluded column list
+     */
+    protected abstract String excludeColumns(TableId tableId);
```

Comment:
I think I would rather this method be named `getSnapshotSelectColumns` or similar.